client/src/containers/AddPostContainer/index.tsx

```diff
@@ -1,30 +1,38 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { uploadPost } from '../../store/addPost/action';
-import AddPostCropper, {IProps} from '../../components/AddPost/AddPostCropper';
+import { uploadPost, setCroppedImageForPost, setDescriptionForPost, resetAddPost } from '../../store/addPost/action';
+import AddPostCropper from '../../components/AddPost/AddPostCropper';
 import {IState} from '../../store/addPost/reducers';
 
 export interface ILocalState {
     addPost: IState;
 }
 
-class AddPostContainer extends React.Component<IProps> {
+class AddPostContainer extends React.Component<any> {
```

Comment:
is this will be another logic? or component is done?

Reply:
No, won't be another logic, component done.